common/lib/xmodule/xmodule/course_module.py

```diff
@@ -294,6 +294,12 @@ class CourseFields(object):
         help=_("Enter the unique identifier for your course's video files provided by edX."),
         scope=Scope.settings
     )
+    facebook_url = String(
+        help=_("Enter the url of the official Facebook page of the course."),
+        default="",
+        display_name=_("Course Facebook Page"),
+        scope=Scope.settings
+    )
```

Comment:
Re-discussed this feature with @lou-wang (!). Here are my revisions.
display_name: Facebook URL
help: Enter the URL for the official course Facebook group. If you provide a URL, the edX mobile app includes a button that students can tap to access the group.